packages/main/src/components/AnalyticalTable/AnalyticalTable.module.css

```diff
@@ -0,0 +1,312 @@
+    border-inline-end: none;
+  }
+}
+
+.tableHeaderBackgroundElement {
+  box-sizing: border-box;
+  position: absolute;
+  inset-inline: 0;
+  height: var(--_ui5wcr-AnalyticalTableHeaderRowHeight);
+  background-color: var(--sapList_HeaderBackground);
+  border-block-start: var(--_ui5wcr-AnalyticalTable-OuterBorderBlock);
+  border-block-end: var(--_ui5wcr-AnalyticalTable-HeaderBorderWidth) solid var(--sapList_HeaderBorderColor);
+}
+
+.tableBodyBackgroundElement {
+  box-sizing: border-box;
+  position: absolute;
+  inset-inline: 0;
+  inset-block-end: 0;
+  border-block-end: 1px solid var(--sapList_TableFooterBorder);
+}
+
+.th {
+  height: var(--_ui5wcr-AnalyticalTableHeaderRowHeight);
+  color: var(--sapList_HeaderTextColor);
+  background-color: var(--sapList_HeaderBackground);
+  border-block-start: var(--_ui5wcr-AnalyticalTable-OuterBorderBlock);
+  border-block-end: var(--_ui5wcr-AnalyticalTable-HeaderBorderWidth) solid var(--sapList_HeaderBorderColor);
+  border-inline-end: 1px solid var(--sapList_BorderColor);
+  padding-inline: 0.5rem;
+  text-align: start;
+  box-sizing: border-box;
+
+  &:hover {
+    background-color: var(--sapList_Hover_Background);
+    color: var(--_ui5wcr-AnalyticalTable-HeaderActiveTextColor);
+  }
+
+  &:active {
+    background-color: var(--sapList_Active_Background);
+    color: var(--_ui5wcr-AnalyticalTable-HeaderActiveTextColor);
+  }
+
+  &[data-column-id='__ui5wcr__internal_highlight_column'] {
+    border-inline-end: 1px solid transparent;
+  }
+
+  &:last-child [data-resizer] {
+    transform: translateX(0px);
+  }
+
+  &:focus {
+    &[data-column-id='__ui5wcr__internal_selection_column'] {
+      border-inline-start: 1px solid transparent;
+    }
+    outline-offset: calc(-1 * var(--sapContent_FocusWidth));
+    outline: var(--sapContent_FocusWidth) var(--sapContent_FocusStyle) var(--sapContent_FocusColor);
+  }
+}
+
+.tbody {
+  position: relative;
+  overflow-x: hidden;
+  overflow-y: auto;
+  scrollbar-width: none;
+  box-sizing: border-box;
+  -ms-overflow-style: none;
+
+  &::-webkit-scrollbar {
+    width: 0 !important;
+    height: 0 !important;
+    display: none !important;
+  }
+}
+
+.alternateRowColor {
+  background-color: var(--sapList_AlternatingBackground);
+}
+
+.tr {
+  position: absolute;
+  inset-block-start: 0;
+  inset-inline-start: 0;
+  width: 100%;
+  color: var(--sapList_TextColor);
+  border-block-end: 1px solid var(--sapList_BorderColor);
+  /* needed for vertical virtualization*/
+  margin-bottom: -1px;
+  box-sizing: border-box;
+  display: flex;
+
+  &:hover:not([data-empty-row]) {
+    background-color: var(--sapList_Hover_Background);
+  }
+
+  &[data-is-selected] {
+    border-block-end: 1px solid var(--sapList_SelectionBorderColor);
+    background-color: var(--sapList_SelectionBackgroundColor);
+  }
+
+  &[data-is-selected]:hover {
+    background-color: var(--sapList_Hover_SelectionBackground);
+  }
+}
+
+.trActive {
+  cursor: pointer;
+}
```

Comment:
this used to be `.trActive:hover` - intended change?

Reply:
Imo it doesn't make sense setting this style only on hover as the cursor is anyway only visible when hovered.